__tests__/app.tests.js

```diff
@@ -3,10 +3,23 @@ const db = require("../db/connection");
 const request = require("supertest");
 const data = require("../db/data/test-data");
 const seed = require("../db/seeds/seed");
+const endpointsJSON = require('../endpoints.json')
 
-beforeAll(() => seed(data));
+beforeEach(() => seed(data));
 afterAll(() => db.end());
 
+describe("/api", () => {
+  test("GET 200: sends an object describing all the available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const {endpoints} = response.body;
+        expect(endpoints).toMatchObject(endpointsJSON)
+          });
```

Comment:
It'll be better if it's a bit more detailed. Although your test will pass, if the content in endpoints.json is accidentally removed, the test will also pass. Thinking about the structure of the endpoints.json. We'd like to test it follows the same structure